README.md

```diff
@@ -45,6 +47,34 @@ PubsubClient.publish(message, 'some-topic') do |result|
 end
 ```
 
+### Subscribing
+
+To subscribe to a topic, a client must first get a handle to the subscriber object. After doing so, a call to `subscriber.listener` will yield two arguments: the data (most clients will only need this) and the full Pub/Sub message (for anything more robust). Optionally, a client can choose to handle exceptions raised by the subscriber.
```

Comment:
You should link to the documentation for `received_message` so clients can get more details.